transform/variant/miss_analysis.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+""" summarize logs to analyze misses  """
+
+# print total imported
+# see README.md for example how to call
+from __future__ import print_function
```

Comment:
We're in python 3 only land now, so this isn't needed.

Reply:
done in next PR (ccle)